inlang/source-code/editor/src/pages/@host/@owner/@repository/components/Gitfloat.tsx

```diff
@@ -292,14 +296,7 @@ export const Gitfloat = () => {
 		}
 	})
 
-	// // trigger refetch of repo when user is logged in and
-	// createEffect(() => {
-	// 	if (lixErrors().some((err) => err.message.includes("401")) && localStorage?.user?.isLoggedIn && repo()) {
```

Comment:
why did you remove this? this was responsible for reloading the page when a token expires